integration-tests/http/__tests__/order-edits/order-edits.spec.ts

```diff
@@ -368,7 +368,7 @@ medusaIntegrationTestRunner({
           )
         ).data.order_preview
 
-        expect(result.summary.current_order_total).toEqual(84)
+        expect(result.summary.current_order_total).toEqual(86.4)
```

Comment:
Item 1 - 12$
Item 2 - 25$
Shipping - 10$

Total before - (25 * 2) + (12 * 2) + 10 = 84

If both item 1 and item 2 have a 10% tax, shouldn't it be:

`(25 * 2 + 5) + (12 * 2 + 2.4) + 10 = 91.4`

Reply:
Item 1 doesn't have taxes, it is created manually on the order using the order module. Since we don't refresh all tax lines, but only the ones generated by added/updated items, the item remains without taxes.

Reply:
Ah wait, I see that the new item that was added will not have a tax as its being built from scratch. So it will be this

`(25 * 2) + (12 * 2 + 2.4) + 10 = 86.4`